lib/jwt/AccessToken.js

```diff
@@ -264,11 +266,17 @@ _.extend(AccessToken.prototype, {
     };
     if (_.isNumber(this.nbf)) { payload.nbf = this.nbf; }
 
+    var header = {
+      cty: 'twilio-fpa;v=1',
+      typ: 'JWT'
+    };
+
+    if (this.region && typeof this.region === 'string') {
```

Comment:
While we still have a lodash as a dependency, I'd prefer the second half of this statement be `_.isString(this.region)` for consistency